tests/e2e-ui-tests/data/workorders.do.js

```diff
@@ -0,0 +1,175 @@
+var params = {
+  WORKER_TCRUDL1: 'wrk1-wor-crudl',
+  WORKER_TCRUDL2: 'wrk2-wor-crudl',
+  WORKFLOW_TCRUDL1: 'wfw1-wor-crudl',
+  WORKFLOW_TCRUDL2: 'wfw2-wor-crudl',
+  WORKORDER_TCREATE: 'create-wor-crudl',
+  WORKORDER_TCANCEL: 'cancel-wor-crudl',
+  WORKORDER_TUPDATE1: 'update1-wor-crudl',
+  WORKORDER_TUPDATE2: 'update2-wor-crudl',
+  WORKORDER_TDELETE: 'delete-wor-crudl',
+  WORKORDER_TSEARCH: 'search-wor-crudl'
+};
+var workorders = {
+  CREATE: {
+    workflow: params.WORKFLOW_TCRUDL1,
+    assignee: params.WORKER_TCRUDL1,
+    title: params.WORKORDER_TCREATE,
+    address: 'Create Street 11',
+    latitude: '412412.2',
+    longitude: '21026.5',
+    startDate: '10/11/2017',
```

Comment:
This is no longer used as start time should be setup on mobile, same with finish time. 
This fields are not needed.